backend/app/services/readers/kinds.py

```
@@ -54,6 +54,50 @@ class KindType(str, Enum):
+    """Resolve direct or entity-derived Team sharing with the unified policy."""
+    from app.schemas.share import MemberRole
+    from app.services.share.team_share_service import team_share_service
+
+    candidates = (
+        db.query(Kind)
+        .filter(
+            Kind.user_id.notin_((0, user_id)),
+            Kind.kind == KindType.TEAM.value,
+            Kind.namespace == namespace,
+            Kind.name == name,
+            Kind.is_active == True,
```

Ruff (0.16.3)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)